src/Compilers/Core/Portable/Diagnostic/Diagnostic.cs

```diff
@@ -281,7 +281,7 @@ internal static Diagnostic Create(DiagnosticInfo info)
         /// <summary>
         /// Returns true if the diagnostic has a source suppression, i.e. an attribute or a pragma suppression.
         /// </summary>
-        public abstract bool IsSuppressed { get; }
+        public virtual bool IsSuppressed { get { throw new NotImplementedException(); } }
```

Comment:
@tvsonar here is the code we use to determine if a type is publically extendable or not

https://github.com/dotnet/roslyn-analyzers/blob/master/src/Roslyn.Diagnostics.Analyzers/Core/DeclarePublicAPIAnalyzer.Impl.cs#L189

For types where this is `false` we don't consider introducing `abstract` methods as a breaking change and prefer them over `virtual` methods.